scripts/cluster_providers/kube.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+function create_cluster(){
+   echo "No need to create cluster since kubeconfig is provided externally"
+   echo "TODO: Setup persistent volume"
+}
+
+function delete_cluster() {
+   echo "Deletion can not be performed on external cluster"
+}
+
+function get_cluster_auth() {
+   echo "export KUBECONFIG=$1"
```

Comment:
can we use the variable name kubeconfig or something for this? how do we make sure this metadata is a kubeconfig?

Reply:
the reason I named it metadata because It can be kind cluster version or can kubeconfig file

Reply:
yup agree, in case of kind its version number and in case of openshift its kubeconfig path. And here we are exporting the variable to `KUBECONFIG`. I am asking, can we put some kind of check before this, this will block the tests to even start. And we should be able to understand the kube path is not set properly.

Reply:
well if provided `KUBECONFIG` is incorrect then eventually test cases would fail. shouldn't that be expected ? before starting the test there is a check which does `kubectl get nodes` with provided cluster auth command. So we would anyways panic on error. @bamachrn